src/FacebookEventForwarder.js

```diff
@@ -49,6 +49,14 @@
             PhoneNumber3: 21,
         },
         SupportedCommerceTypes = [],
+        //  Standard FB Event Names from https://developers.facebook.com/docs/facebook-pixel/reference#standard-events
+        PAGE_VIEW_EVENT_NAME = 'PageView';
+        ADD_TO_CART_EVENT_NAME = 'AddToCart';
+        REMOVE_FROM_CART_EVENT_NAME = 'RemoveFromCart';
+        Add_TO_WISHLIST_EVENT_NAME = 'AddToWishlist';

```

Comment:
Why `Add_TO_WISHLIST_EVENT_NAME` and not `ADD_TO_WISHLIST_EVENT_NAME`?

Reply:
Good catch.